src/scale.ts

```diff
@@ -273,6 +275,7 @@ export function scaleTypeSupportProperty(scaleType: ScaleType, propName: string)
     case 'domain':
     case 'range':
     case 'scheme':
+    case 'interpolate':
```

Comment:
Pro-tip:   Sometimes, you might want to add some logic that's not well thought to make the code compile to test other parts.  That's totally reasonable, except that you should always add `// FIXME: correct this logic` to remind yourself that you haven't thought about it and make sure to fix it later.